### PR TITLE
fix svace issue. @open sesame 6/14 14:17

### PR DESCRIPTION
**Pull request by yeonykim2**
error param should be paased to 'json_parser_load_from_data' since error_report acess to error message.

### CODE REVIEW

**Review by jaeyun-jung — APPROVED** [review]

**Review by niley7464 — APPROVED** [review]
LGTM :)

File: c/src/ml-api-service-agent-client.c [hunk 1]
```diff
@@ -52,7 +52,7 @@ _build_ml_opt_from_json_cstr (const gchar * jcstring, ml_option_h * opt)
     return ML_ERROR_OUT_OF_MEMORY;
   }
 
-  if (!json_parser_load_from_data (parser, jcstring, -1, NULL)) {
+  if (!json_parser_load_from_data (parser, jcstring, -1, &err)) {
```
Comment (jaeyun-jung) [inline c/src/ml-api-service-agent-client.c:55]: suggestion:
```
_ml_error_report ("Failed to parse the json string. %s", err ? err->message : "unknown error");
```

Comment (yeonykim2) [reply c/src/ml-api-service-agent-client.c:55]: thx updated.

**Review by wooksong — APPROVED** [review]
LGTM:)

**Review by songgot — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**